rfc6962-bis.xml

```diff
@@ -474,7 +474,7 @@ d0 d1   d2 d3           d0 d1   d2 d3  d4 d5</artwork>
       <section title="Log ID" anchor="log_id">
         <figure>
           <preamble>
-            Each log is uniquely identified by an OID (specified in the log's metadata). A log's operator MUST either allocate the OID themselves or request an OID from one of the two Log ID Registries (see <xref target="log_id_registry1"/> and <xref target="log_id_registry2"/>). Various data structures include the DER encoding of this OID, excluding the ASN.1 tag and length bytes, in an opaque vector:
+            Each log is identified by an OID (specified in the log's metadata) which MUST be unique. A log's operator MUST either allocate the OID themselves or request an OID from one of the two Log ID Registries (see <xref target="log_id_registry1"/> and <xref target="log_id_registry2"/>). Various data structures include the DER encoding of this OID, excluding the ASN.1 tag and length bytes, in an opaque vector:
```

Comment:
I prefer the original "uniquely identified by an OID" text.

"an OID...which MUST be unique" doesn't really say anything.  Each OID is unique by definition!  e.g. there is only 1 OID that has the value 1.2.3.4.

The important point we need to make is that each OID MUST only be used as the Log ID for 1 log.  I think "is uniquely identified" expresses this.  That said, if you feel that it needs to be expressed more verbosely, how about this...

"Each log is identified by an OID, which is specified in the log's metadata and which MUST NOT be used to identify any other log."